core/utils/constants.php

```diff
@@ -1,16 +0,0 @@
-<?php
```

Comment:
I wonder why we added this and never used it? 🤔

Reply:
Changes were made and this wasn't removed...